cmd/mdatagen/README.md

```diff
@@ -33,7 +33,7 @@ Detailed information about the schema of `metadata.yaml` can be found in [metada
 
 The `go:generate mdatagen` directive is usually defined in a `doc.go` file in the same package as the component, for example:
 ```go
-//go:generate mdatagen metadata.yaml
+//go:generate env GITHUB_PROJECT=open-telemetry/opentelemetry-collector mdatagen metadata.yaml
```

Comment:
i would prefer this be configurable in metadata.yaml, it avoids having configuration in two places

Reply:
See https://github.com/atoulme/opentelemetry-collector/pull/1